src/Datadog.Trace.ClrProfiler.Managed/Integrations/AdoNet/SqlCommandIntegration.cs

```diff
@@ -10,15 +10,28 @@
 namespace Datadog.Trace.ClrProfiler.Integrations.AdoNet
 {
     /// <summary>
-    /// Instrumentation wrappers for SqlCommand.
+    /// Instrumentation wrappers for <c>System.Data.SqlClient.SqlCommand</c>
+    /// and <c>Microsoft.Data.SqlClient.SqlCommand</c>.
     /// </summary>
     public static class SqlCommandIntegration
     {
+        private const string Major1 = "1";
+        private const string Major2 = "2";
         private const string Major4 = "4";
         private const string Major5 = "4";
```

Comment:
Feel free to disregard, but since SqlCommand lives in `System.Data.SqlClient, Version=4.6.1.1, PublicKeyToken=b03f5f7f11d50a3a` for .NET 5, we can revert the changes that use Major5. After all, it never got incremented anyway 😄

Reply:
Removed `Major5` in 29d89177